### PR TITLE
Add GitHub issue template

### PR DESCRIPTION
**Pull request by imiric**
Mostly copied from the k6 repo.

### CODE REVIEW

**Review by ankur22 — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+contact_links:
+  - name: k6 Community Forum
+    url: https://community.k6.io/
```
Comment (inancgumus) [inline .github/ISSUE_TEMPLATE/config.yml:4]:
```suggestion
  - name: xk6-browser Community Forum
    url: https://community.k6.io/c/xk6-browser/14
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+    about: Please ask and answer questions here.
+  - name: k6 Documentation
+    url: https://github.com/grafana/k6-docs
```
Comment (inancgumus) [inline .github/ISSUE_TEMPLATE/config.yml:7]:
```suggestion
  - name: xk6-browser Documentation
    url: https://k6.io/docs/javascript-api/xk6-browser/
```

**Review by inancgumus — CHANGES_REQUESTED** [review]
LGTM with some suggestions.

File: .github/ISSUE_TEMPLATE/config.yml (hunk 2 above)
Comment (imiric) [reply .github/ISSUE_TEMPLATE/config.yml:7]: This link is meant for adding documentation issues, so linking to the k6-docs repo is correct.

**Review by inancgumus — APPROVED** [review]

**Pull request merged**